### PR TITLE
Minor non-semantic refactors of `ExternalTexturesFlutterActivity`

### PR DESCRIPTION
**Pull request by matanlurey**
Just getting rid of some IDE warnings/diagnostics before sending bigger PRs.

### CODE REVIEW

File: testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ExternalTextureFlutterActivity.java [hunk 1]
```diff
@@ -92,7 +94,9 @@ public void waitUntilFlutterRendered() {
     super.waitUntilFlutterRendered();
 
     try {
-      firstFrameLatch.await(10, java.util.concurrent.TimeUnit.SECONDS);
+      if (!firstFrameLatch.await(10, java.util.concurrent.TimeUnit.SECONDS)) {
+        throw new RuntimeException("Timeout waiting for firstFrameLatch to signal");
```
Comment (gaaclarke) [inline testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ExternalTextureFlutterActivity.java:98]: There is a class for timeouts: https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/TimeoutException.html

File: testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ExternalTextureFlutterActivity.java [hunk 2]
```diff
@@ -127,8 +131,9 @@ private MediaExtractor createMediaExtractor() {
     // -profile:v main -level:v 5.2 -t 1 -r 1 -vf scale=192:256 -b:v 1M sample.mp4
     try {
       MediaExtractor extractor = new MediaExtractor();
-      AssetFileDescriptor afd = getAssets().openFd("sample.mp4");
-      extractor.setDataSource(afd.getFileDescriptor(), afd.getStartOffset(), afd.getLength());
+      try (AssetFileDescriptor afd = getAssets().openFd("sample.mp4")) {
```
Comment (gaaclarke) [inline testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ExternalTextureFlutterActivity.java:134]: Does this do something without a catch or finally block?

Comment (gaaclarke) [reply testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ExternalTextureFlutterActivity.java:134]: oops didn't see it had a resource, lgtm

File: testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ExternalTextureFlutterActivity.java (hunk 1 above)
Comment (matanlurey) [reply testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/ExternalTextureFlutterActivity.java:98]: It's a checked exception though, I don't think we actually want our callers to catch anything (it's not actionable).

**Review by gaaclarke — APPROVED** [review]
lgtm!

**Pull request merged**